src/hotspot/share/gc/shared/memAllocator.hpp

```diff
@@ -25,9 +25,9 @@
 #ifndef SHARE_GC_SHARED_MEMALLOCATOR_HPP
 #define SHARE_GC_SHARED_MEMALLOCATOR_HPP
 
-#include "gc/shared/collectedHeap.hpp"
 #include "memory/memRegion.hpp"
 #include "oops/oopsHierarchy.hpp"
+#include "runtime/thread.hpp"
```

Comment:
`utilities/globalDefinitions.hpp` for `HeapWord` is missing.

Reply:
Fixed.